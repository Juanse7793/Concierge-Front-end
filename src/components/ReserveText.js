import React from 'react';

const ReserveText = () => (
  <p>
    Hello and welcome to the
    {' '}
    <b>CONCIERGE EVENT RESERVATION.</b>
    {' '}
    <br />
    Please enter the dates you would like to attend
    and the city you will be traveling from
    in the form below to book
    <br />
    <b><i>an exclusive, all inclusive</i></b>
    <br />
    trip to the event.
    <br />
    <br />
    {' '}
    <b>CONCIERGE</b>
    {' '}
    gives you the ultimate worry-free traveling experience,
    knowing all your needs will have been taken care of by
    {' '}
    <b>CONCIERGE.</b>
  </p>
);

export default ReserveText;
