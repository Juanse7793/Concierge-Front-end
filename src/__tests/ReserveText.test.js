import renderer from 'react-test-renderer';
import ReserveText from '../components/ReserveText';

it('matches snapshot', () => {
  const component = renderer.create(
    <ReserveText />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
