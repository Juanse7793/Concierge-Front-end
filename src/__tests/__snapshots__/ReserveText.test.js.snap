// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<p
  className="black-glow"
>
  Hello and welcome to the
  <b>
     CONCIERGE EVENT RESERVATION.
  </b>
  <br />
  Please enter the dates you would like to attend and the city you will be traveling from in the form below to book
  <br />
  <b>
    <i>
      an exclusive, all inclusive
    </i>
  </b>
  <br />
  trip to the event.
  <br />
  <br />
  <b>
    CONCIERGE
  </b>
   
  gives you the ultimate worry-free traveling experience, knowing all your needs will have been taken care of by
  <b>
     CONCIERGE.
  </b>
</p>
`;
